<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PVPGame</class>
 <widget class="QDialog" name="PVPGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1111</width>
     <height>901</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
    <item>
     <widget class="QGraphicsView" name="Game_View">
      <property name="font">
       <font>
        <pointsize>21</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="P1">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255,130,71);
border-radius:20px</string>
        </property>
        <property name="text">
         <string>P1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ScoreLabel">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>21</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255,106,106);
border-radius:20px;</string>
        </property>
        <property name="text">
         <string>Score:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="SpeedLabel">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>21</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(132,112,250);
border-radius:20px;
</string>
        </property>
        <property name="text">
         <string>Speed:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LifeLabel">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>21</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(124,205,124);
border-radius:20px;
font-family:Microsoft YaHei</string>
        </property>
        <property name="text">
         <string>HP:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="P2">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255,130,71);
border-radius:20px;</string>
        </property>
        <property name="text">
         <string>P2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ScoreLabel_2">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>21</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255,106,106);
border-radius:20px;</string>
        </property>
        <property name="text">
         <string>Score:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="SpeedLabel_2">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>21</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(132,112,250);
border-radius:20px;
</string>
        </property>
        <property name="text">
         <string>Speed:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LifeLabel_2">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>21</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(124,205,124);
border-radius:20px</string>
        </property>
        <property name="text">
         <string>HP:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255,99,71);
border-radius:20px</string>
        </property>
        <property name="text">
         <string>Start!</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/icons/start.png</normaloff>:/images/icons/start.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pausebutton">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255,99,71);
border-radius:20px</string>
        </property>
        <property name="text">
         <string>Pause!</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/icons/pause.png</normaloff>:/images/icons/pause.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
